FEMElements/FEMElements.sln

```diff
@@ -1,25 +1,9 @@
 ﻿
```

Comment:
Remove solution file from your pull request since it does not reflect any actual code changes.